utils/testutils/mocked_http_service.go

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2016-present Mattermost, Inc. All Rights Reserved.
```

Comment:
Maybe each service musts contain its own testutils?

Reply:
I'm 0/5 about that. I put them in `utils/testutils` since I want to avoid the problem files like `app/apptestlib.go` where they're package-specific and they're built with non-test code, but I don't think it's absolutely necessary to have a separate package for each service's mock implementation.

Thoughts? We can always move them again if necessary.